INSTALL

```diff
@@ -8,11 +8,15 @@ They freely distributed at address:
      http://caml.inria.fr/ocaml/
      http://pauillac.inria.fr/~ddr/camlp5/
 
+0. prerequisites:
+
+$ opam install zarith jingoo ppx_import uri uunf camlzip markup unidecode calendars alcotest
```

Comment:
why not to use something like opam install ./geneweb.opam  with adequate parameter to avoid to provide dependency list ?

Reply:
haha good point.  I read the INSTALL file and proceeded to get lost, didn't even look for an OPAM file.